backend/src/services/payments.js

```diff
@@ -0,0 +1,63 @@
+const axios = require('axios');
+const crypto = require('crypto');
+const db = require('../db');
+
+const tokenUrl = 'https://payform.bambora.com/pbwapi/auth_payment';
+const bamboraApi = 'https://payform.bambora.com/pbwapi/token/';
+
+const secret = '267e417772b5e6eab74925e0b244865e';
```

Comment:
If these are the credentials gotten from Bambora, they cannot be stored in version control at all. Instead they need to be made available via the environmental variables. In that case, they also need to be resetted on Bambora and replaced with new ones since once credentials are in Git, you can't assume they could be taken away without possibility of breach.